project/Admin.cs

```diff
@@ -21,6 +23,8 @@ public string Name
             }
         }
 
+        public List<Timeslot> ManagedTimeslots { get; private set; } = new();
```

Comment:
If a getter is called, we need to return a copy of the List (applies to all reverse connections that involve a data structure of some kind)...

Reply:
will update the getter to return a new list containing the current elements, ensuring encapsulation and immutability